iterators.tex

```diff
@@ -623,31 +650,186 @@
 \textit{p}).
 \exitexample
 
-\indexlibrary{\idxcode{Iterator}}%
-\begin{codeblock}
+{\color{newclr}
+\rSec2[iterators.sizediterator]{Concept SizedIterator}
+\pnum
+The \tcode{SizedIterator} concept refines \tcode{Iterator} with the
+capability to determine the distance between two iterators that denote
+elements of the same range in constant time with the \tcode{-} operator.
+
+\indexlibrary{\idxcode{SizedIterator}}%
+\begin{itemdecl}
+  template <class>
+  constexpr bool disable_sized = false;
```

Comment:
Oh, you're saying that it's the user's responsibility to property constrain `op-` for sentinels. Hrm. I'm still rather uncomfortable.